core/src/main/java/com/sequenceiq/cloudbreak/service/identitymapping/AwsMockAccountMappingService.java

```diff
@@ -44,7 +47,14 @@ public AwsMockAccountMappingService(CloudPlatformConnectors cloudPlatformConnect
 
     public Map<String, String> getUserMappings(String region, CloudCredential credential) {
         String accountId = getAccountId(region, credential);
-        return replaceAccountId(MOCK_IDBROKER_USER_MAPPINGS, accountId);
+        String partition = getPartition(credential);
+        Map<String, String> mapping = replaceAccountId(MOCK_IDBROKER_USER_MAPPINGS, accountId);
```

Comment:
please refactor the multiple placeholder substitution by using some template handler, like https://commons.apache.org/proper/commons-text/apidocs/org/apache/commons/text/StringSubstitutor.html